### PR TITLE
UI: Various audio mixer adjustments and clean up

### PR DESCRIPTION
**Pull request by Warchamp7**
### Description
These commits make a few adjustments or fixes to the audio mixer.

1. Group the buttons together in horizontal layout for better alignment
2. Cleans up some spacing and sizing of audio mixer widgets
3. Remove wheel events from the absolute-slider event filter. This is still appropriately handled by the slider-ignorewheel widget
4. macOS-specific fix. May be able to squash
5. Center widgets in vertical layout

Before:
image

After:
image

Before:
image

After:
image

### Motivation and Context
Fix and clean up audio mixer changes.

### How Has This Been Tested?
Tried scrolling the audio mixer while hovering a volume meter or slider

### Types of changes
- Bug fix (non-breaking change which fixes an issue) -->

- Tweak (non-breaking change to improve existing functionality) -->

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Review by Lain-B — APPROVED** [review]

File: UI/volume-control.cpp [hunk 1]
```diff
@@ -362,23 +369,24 @@ VolControl::VolControl(OBSSource source_, bool showConfig, bool vertical)
...
+			new QSpacerItem(0, 0, QSizePolicy::Minimum,
+					QSizePolicy::MinimumExpanding));
+		buttonLayout->addWidget(mute);
 
 		mainLayout->addItem(textLayout);
-		mainLayout->addWidget(meterFrame);
-		mainLayout->addItem(botLayout);
+		mainLayout->addItem(controlLayout);
+		controlLayout->addItem(buttonLayout);
+		controlLayout->addWidget(meterFrame);
```
Comment (RytoEX) [inline UI/volume-control.cpp:389]: I would generally expect a sub-layout to have all of its items and widgets added before adding the sub-layout to another layout.
```suggestion
		controlLayout->addItem(buttonLayout);
		controlLayout->addWidget(meterFrame);
		mainLayout->addItem(controlLayout);
```

**Review by RytoEX — CHANGES_REQUESTED** [review]
I've made a suggestion, but I could be convinced to leave it as is.

File: UI/volume-control.cpp (hunk 1 above)
Comment (Warchamp7) [reply UI/volume-control.cpp]: Done

**Review by RytoEX — APPROVED** [review]
LGTM

**Pull request merged**